.circleci/unittest/linux/scripts/setup_env.sh

```diff
@@ -56,17 +56,25 @@ cat "${this_dir}/environment.yml"
 if [[ $OSTYPE == 'darwin'* ]]; then
   PRIVATE_MUJOCO_GL=glfw
 else
-  conda install -y -c conda-forge mesa
-  conda install -y -c conda-forge mesalib
   PRIVATE_MUJOCO_GL=osmesa
 fi
 
 export MUJOCO_GL=$PRIVATE_MUJOCO_GL
 conda env config vars set MUJOCO_PY_MUJOCO_PATH=$root_dir/.mujoco/mujoco210 \
   DISPLAY=unix:0.0 \
   MJLIB_PATH=$root_dir/.mujoco/mujoco-2.1.1/lib/libmujoco.so.2.1.1 \
-  LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$root_dir/.mujoco/mujoco210/bin \
+  LD_LIBRARY_PATH=$root_dir/.mujoco/mujoco210/bin \
   SDL_VIDEODRIVER=dummy \
-  MUJOCO_GL=$PRIVATE_MUJOCO_GL
+  MUJOCO_GL=$PRIVATE_MUJOCO_GL\
+  PYOPENGL_PLATFORM=osmesa
```

Comment:
In my PR I used egl, can we stick to that?

Reply:
sure let's do that.